test/integration/test_expid.py

```diff
@@ -110,3 +134,127 @@ def test_copy_minimal(has_min_yaml, autosubmit_exp, autosubmit):
+        platform_name,
+        minimal_configuration=True
+    )
+
+    # capture the platform using the "describe"
+    describe = autosubmit.describe(experiment_id)
+    hpc_result  = describe[4].lower()
+
+    assert hpc_result == platform_name
+
+@pytest.mark.parametrize("fake_hpc, expected_hpc", [
+    ("mn5", "mn5"),
+    ("", "local"), ])
+def test_create_expid_flag_hpc(fake_hpc: str, expected_hpc: str, autosubmit: Autosubmit) -> None:
+    """Create expid using the flag -H. Defining a value for the flag and not defining any value for that flag.
+
+    .. code-block:: console
+
+        autosubmit expid -H ithaca -d "experiment is about..."
+        autosubmit expid -H "" -d "experiment is about..."
+
+    :param fake_hpc: The value for the -H flag (hpc value).
+    :type fake_hpc: str
+    :param expected_hpc: The value it is expected for the variable hpc.
+    :type expected_hpc: str
+    :param autosubmit: Autosubmit interface that instantiate with no experiment.
+    :type autosubmit: Autosubmit
+
+    :return: None
+    """
+    # create default expid with know hpc
+
+    experiment_id = autosubmit.expid(
+        'experiment',
+        fake_hpc,
+        minimal_configuration=True
+    )
+
+    # capture the platform using the "describe"
+    describe_experiment = autosubmit.describe(experiment_id)
+    hpc_result_experiment = describe_experiment[4].lower()
+
+    assert hpc_result_experiment == expected_hpc
+
+
+@pytest.mark.parametrize("fake_hpc, expected_hpc", [
+    ("mn5", "mn5"),
+    ("", "local"),
+])
+def test_copy_expid(fake_hpc: str, expected_hpc: str, autosubmit: Autosubmit) -> None:
+    """Copy an experiment without indicating which is the new HPC platform
+
+    .. code-block:: console
+
+        autosubmit expid -y a000 -d "experiment is about..."
+
+    :param fake_hpc: The value for the -H flag (hpc value).
+    :type fake_hpc: str
+    :param expected_hpc: The value it is expected for the variable hpc.
+    :type expected_hpc: str
+    :param autosubmit: Autosubmit interface that instantiate with no experiment.
+    :type autosubmit: Autosubmit
+
+    :return: None
+    """
+    # create default expid with know hpc
+
+    original_id = autosubmit.expid(
+        'original',
+        fake_hpc,
+        minimal_configuration=True
+    )
+
+    copy_id = autosubmit.expid('copy', "", original_id)
+
+    # capture the platform using the "describe"
+    describe_copy = autosubmit.describe(copy_id)
+    hpc_result_copy = describe_copy[4].lower()
+
+    assert hpc_result_copy == expected_hpc
+
+
+# copy expid with specific hpc should not change the hpc value
+# autosubmit expid -y a000 -h local -d "experiment is about..."
+def test_copy_expid_no(autosubmit: Autosubmit) -> None:
+    """Copy an experiment with specific HPC platform
+
+    .. code-block:: console
+
+        autosubmit expid -y a000 -h local -d "experiment is about..."
+
+    :param autosubmit: Autosubmit interface that instantiate with no experiment.
```

Comment:
s/instantiate/instantiates or simply "Autosubmit instance with no experiments."? Not important